web/src/compose_recipient.js

```diff
@@ -108,25 +108,38 @@ export function update_on_recipient_change() {
     update_narrow_to_recipient_visibility();
 }
 
-export function check_stream_posting_policy_for_compose_box(stream_name) {
-    const stream = stream_data.get_sub_by_name(stream_name);
-    if (!stream) {
-        return;
+export function get_posting_policy_error_message() {
```

Comment:
I would prefer to structure this function with a clear outer if/else for direct messages vs. stream messages; that'll prove a lot nicer as we add more private message policies.

Reply:
I'm not getting you clearly, Are you suggesting to pass `msg_type` as arg or restructure if/else inside function?

Reply:
Just the if/else structure.

Reply:
Ooh okay got it !